lib/full-site-editing/edit-site-page.php

```diff
@@ -31,6 +31,15 @@ function gutenberg_is_edit_site_page( $page ) {
 	return 'appearance_page_gutenberg-edit-site' === $page;
 }
 
+/**
+ * Checks whether the provided page is the templates list page.
+ *
+ * @return bool True for Site Editor pages, false otherwise.
+ */
+function gutenberg_is_edit_site_list_page() {
+	return isset( $_GET['postType'] ) && ! isset( $_GET['postId'] );
```

Comment:
Could you not use get_screen or similar.

Reply:
I'm not sure what you mean here? Sorry I'm not really familiar with PHP 😅 .

Reply:
I don't think we can. These are query arguments on the Site Editor page.